src/main/java/org/gogame/server/domain/entities/GameEntity.java

```diff
@@ -0,0 +1,32 @@
+package org.gogame.server.domain.entities;
+
+import jakarta.persistence.*;
+import lombok.*;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Entity
+@Table(name = "game")
+public class GameEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "game_id_seq")
+    @SequenceGenerator(name = "game_id_seq", allocationSize = 1)
+    @Column(nullable = false, unique = true)
+    private Long game_id;
+
+
+    @ManyToOne
+    @JoinColumn(name = "winner_id", referencedColumnName = "user_id")
+    private UserEntity winner;
+
+    @ManyToOne
+    @JoinColumn(name = "user_black_id", referencedColumnName = "user_id")
+    private UserEntity user_black;
+
+    @ManyToOne
+    @JoinColumn(name = "user_white_id", referencedColumnName = "user_id")
+    private UserEntity user_white;
```

Comment:
any reason for these being snake_case instead of camelCase?

Reply:
Not an issue, Spring Data JPA uses property name as table column name by default, unless other name have not specified in `@Column()`

Reply:
isn't there any way to have the names camel case and make spring convert them into snake case when doing sql?

Reply:
I've seen in offical docs that Spring uses camelCase for entity properties. You were right